src/table.rs

```diff
@@ -65,31 +70,146 @@ pub struct Table {
     inner: Arc<TableInner>,
 }
 
+/// `AsRef<TableInner>` is only used in `init_biggest_and_smallest`
+/// to construct a table iterator from `&TableInner`.
+impl AsRef<TableInner> for TableInner {
+    fn as_ref(&self) -> &TableInner {
+        self
+    }
+}
+
 impl TableInner {
     /// Create an SST from bytes data generated with table builder
-    pub fn create(_path: &Path, _data: Bytes, _opt: Options) -> Result<TableInner> {
-        unimplemented!()
+    fn create(path: &Path, data: Bytes, opt: Options) -> Result<TableInner> {
+        let mut f = fs::OpenOptions::new()
+            .create_new(true)
+            .read(true)
+            .write(true)
+            .open(path)?;
+        f.write(&data)?;
+        // TODO: pass file object directly to open
+        drop(f);
```

Comment:
// TODO: sync write

Reply:
Here we dropped `f`, so I guess table writing should have been synchronized when dropping (in current situation). Whatever, I still added this todo.

Reply:
> A successful close does not guarantee that the data has been successfully saved to disk, as the kernel defers writes. It is not common for a file system to flush the buffers when the stream is closed. If you need to be sure that the data is physically stored use fsync(2). (It will depend on the disk hardware at this point.)

No, file will not be synced when dropped. We have met the issue several times that data got lost without syncing them explicitly.